src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+        for (int i = 0; i < 6; i++) {
+            if (i < 3) {
+                figureSupplier.getRandomFigure().printInfo();
+            } else {
+                figureSupplier.getDefaultFigure().printInfo();
+            }
+        }
```

Comment:
task condition is not implemented:
"In the main() method we need to create an array of figures (the size of the array can be 3 or 6, it doesn't matter). The first half of figures in this array should be generated with random parameters."